### PR TITLE
Faq backup update

### PR DESCRIPTION
**Pull request by MohamedZaghdoudi**
This PR resolves the enhancement request #1319 "Update FAQ https://www.coronawarn.app/en/faq/#backup".

The section of https://www.coronawarn.app/de/faq/ has more information about the  vaccination certificate 

image

### CODE REVIEW

**Comment by brianebeling** [comment]
@MohamedZaghdoudi The last entry is empty, is that only in the screenshot?

**Comment by MohamedZaghdoudi** [comment]
@brianebeling thanks for your feedback. No sorry i was not seeing it. I will I will correct it right away.

**Comment by MikeMcC399** [comment]
The last entry is to a glossary entry. It could easily be that the duplicate mechanism does not work with glossary items.

**Comment by MikeMcC399** [comment]
@brianebeling 

- Can you indicate whether you want to proceed with https://github.com/corona-warn-app/cwa-website/pull/3239? https://www.coronawarn.app/en/faq/results/#migration_Apple is modified and marked as obsolete there.
- If that is accepted then #migration_Apple should be removed from this backup PR as it is no longer relevant.

**Comment by MikeMcC399** [comment]
@MohamedZaghdoudi 

You seem to have removed not only the glossary entry, but also the one before it. Was that intended?

**Comment by MohamedZaghdoudi** [comment]
@MikeMcC399 Yes, you are right. I have added again.

**Comment by MikeMcC399** [comment]
Please rebase this PR on the master branch. There are changes in it which should not be there.

File: src/data/faq_de.json [hunk 1]
```diff
@@ -874,7 +874,7 @@
...
-                                    "<b>Wichtig:</b> Prüfen Sie bitte die lokalen Verordnungen der Bundesländer."
+                                    "<b>Wichtig:</b> Prüfen Sie bitte die lokalen Verordnungen der Bundesländer. Berlin beispielsweise verlangt, dass der Nachweis der Impfung gegen oder Genesung von SARS-CoV2 digital verifizierbar sein muss (Siehe <a href='https://www.berlin.de/corona/massnahmen/verordnung/' target='_blank' rel='noopener noreferrer'>https://www.berlin.de/corona/massnahmen/verordnung/</a> §&nbsp;8a Absatz&nbsp;2 Ziffer&nbsp;4). Das heißt, dass Sie gegebenenfalls ein entsprechendes digitales Zertifikat der EU entweder in der Corona-Warn-App, der CovPass-App oder als Papierausdruck zur Überprüfung vorzulegen haben."
```
Comment (MikeMcC399) [inline src/data/faq_de.json:877]: The reference to Berlin has nothing to do with backups and reverts the article to the version before PR https://github.com/corona-warn-app/cwa-website/pull/3232 was merged.
This (and other unrelated changes) should not be part of this branch / PR.

**Comment by MohamedZaghdoudi** [comment]
@MikeMcC399 i will start a new PR. That will be better.

**Comment by MikeMcC399** [comment]
- Since you have opened the new PR https://github.com/corona-warn-app/cwa-website/pull/3260 I suggest that you should close this one if your intention is to replace it.

**Comment by brianebeling** [comment]
Closing this because of the duplication.

**Comment by MikeMcC399** [comment]
Please delete the branch https://github.com/corona-warn-app/cwa-website/tree/faq_backup_update from this closed PR.
